### PR TITLE
Identity cleanup

### PR DESCRIPTION
**Pull request by bshyong**
- Fixes a bug with identity requests
- Cleans up old signin code
- Connected to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/4856, maintains use of IDme signin URL for in progress form saving

@patrickvinograd please make sure this gets merged to `kudos-identity` before that branch is deployed to staging

### CODE REVIEW

File: src/js/signin/containers/Main.jsx [hunk 1]
```diff
@@ -45,11 +49,14 @@ class Main extends React.Component {
     this.loginUrlRequest = getLoginUrls(this.props.onUpdateLoginUrls);
   }
 
-  getVerifyUrl() {
-    if (!this.props.login.currentlyLoggedIn) return;
+  getVerifyUrl(forceRequest) {
+    if (!forceRequest && !this.props.login.currentlyLoggedIn) return;
```
Comment (U-DON) [inline src/js/signin/containers/Main.jsx]: Why was this necessary?

**Review by U-DON — APPROVED** [review]
LGTM. Might want @jbalboni to verify the SiP stuff.

File: src/js/common/helpers/login-helpers.js [hunk 2]
```diff
@@ -111,13 +99,13 @@ export function getLoginUrls(onUpdateLoginUrls) {
...
 
-export function handleLogin(loginUrl, onUpdateLoginUrl) {
+export function handleLogin(loginUrl) {
   window.dataLayer.push({ event: 'login-link-clicked' });
   if (loginUrl) {
     window.dataLayer.push({ event: 'login-link-opened' });
     const receiver = window.open(`${loginUrl}&op=signin`, '_blank', 'resizable=yes,scrollbars=1,top=50,left=500,width=500,height=750');
     receiver.focus();
-    return getLoginUrl(onUpdateLoginUrl);
+    return null;
```
Comment (U-DON) [inline src/js/common/helpers/login-helpers.js]: Should this not be `return getLoginUrls(onUpdateLoginUrls)`? This is a change that's causing some unit test failures in `LoginModal`.

Comment (bshyong) [reply src/js/common/helpers/login-helpers.js]: What would be the purpose of calling getLoginUrls again? I can look at the tests but getting rid of this second call didn't break any of the signin flows. 

We already have a URL and open the modal with that URL in this method so I'm not sure what the purpose of the second call would be

File: src/js/login/reducers/login.js [hunk 3]
```diff
@@ -39,9 +37,7 @@ function loginStuff(state = initialState, action) {
     case UPDATE_LOGGEDIN_STATUS:
       return _.set('currentlyLoggedIn', action.value, state);
 
-    case UPDATE_LOGIN_URL:
-      return _.set('loginUrl', appendQuery(action.value, { clientId: action.gaClientId }), state);
-
+    // being explicit here to avoid confusion
```
[the remark this answers is not in the document]
Comment (bshyong) [reply src/js/login/reducers/login.js]: The duplicated appendQuery code in the reducer for each type of URl

File: src/js/signin/containers/Main.jsx (hunk 1 above)
Comment (bshyong) [reply src/js/signin/containers/Main.jsx]: This is so that the method can be properly called in willReceiveProps if the loggedIn prop changes due to someone logging in, and if somebody lands on the verify page and does a page refresh
